config.ini

```diff
@@ -35,7 +35,7 @@
     "pulse_ramp_time" : "10.0e-6",
     "gpio_bank" : "RXA",
     "tr_window_time" : "60e-6",
-    "agc_signal_read_delay" : "0"
+    "agc_signal_read_delay" : "0",
```

Comment:
unfortunately the docs file for config table is not on this branch but will need to add this to the .rst file describing the config options when merged.